io.github.arunsivaramanneo.GPUViewer.yml

```diff
@@ -1,9 +1,10 @@
 id: io.github.arunsivaramanneo.GPUViewer
-runtime: org.freedesktop.Platform
-sdk: org.freedesktop.Sdk
-runtime-version: "18.08"
+runtime: org.gnome.Platform
```

Comment:
Switching to Gnome runtime can actually increase maintenance work, since it has shorter life-cycle. IMO reusing pygobject isn't worth it.

Reply:
Alright then, I'll switch back.